indico/modules/events/registration/clone.py

```diff
@@ -22,47 +22,85 @@
 from indico.modules.events.registration.models.forms import RegistrationForm
 from indico.modules.events.registration.models.form_fields import RegistrationFormFieldData
 from indico.modules.events.registration.models.items import RegistrationFormItem, RegistrationFormSection
+from indico.modules.events.registration.models.registrations import Registration, RegistrationData
 from indico.util.i18n import _
 
 from MaKaC.conference import EventCloner
 
 
 class RegistrationFormCloner(EventCloner):
+    def __init__(self, *args, **kwargs):
+        super(RegistrationFormCloner, self).__init__(*args, **kwargs)
+        self.field_data_dict = {}
+        self._clone_with_registrations = False
+
     def get_options(self):
         if not is_feature_enabled(self.event, 'registration'):
             return {}
         enabled = bool(self._find_registration_forms().count())
-        return {'form': (_('Registration forms'), enabled, False)}
+        return {
+            'form': (_('Registration forms'), enabled, False),
+            'registrations': (_('Registrations'), enabled, False)
```

Comment:
I would leave it as registrations, otherwise we should make a clearer distinction between *Participants* and *Registrations* everywhere else. *Participants* is a registration form and we are not hiding it. The only thing we added is the side menu and the page to quickly create one. Other than that we show *Registration* everywhere.